packages/protocol/contracts/governance/Validators.sol

```diff
@@ -483,6 +520,7 @@ contract Validators is IValidators, Ownable, ReentrancyGuard, Initializable, Usi
     require(voters[account] == address(0));
     uint256 weight = getAccountWeight(account);
     require(weight > 0);
+    totalVotes = totalVotes.add(weight);
```

Comment:
Looks like you branched off of your electoral threshold PR, adding some extraneous diffs to this one.